backend/src/prompts/templates/intent.j2

```diff
@@ -0,0 +1,39 @@
+You are an expert in determining the intent behind a user's question.
+
+The question is: 
+{{ question }}
+
+The task is to comprehend the intention and format of the question. Based on this understanding, the following query must be formulated to extract the necessary data from a database, which can then be used to address the question.

```

Comment:
Good question. My guess is that it is helping guide the llm to produce a certain "shaped" response instead of just passing back the intention. But, we have added the format prompt, so I'm going to remove "format" here and test. If it still works reliably, will remove